pkg/frontend/validate.go

```diff
@@ -78,7 +78,33 @@ func (f *frontend) validateOpenShiftUniqueKey(ctx context.Context, doc *api.Open
 // prevent mischief
 var rxKubernetesString = regexp.MustCompile(`(?i)^[-a-z0-9.]{0,255}$`)
 
+func validatePermittedClusterwideObjects(groupKind string) bool {
```

Comment:
This now requires SREs to suffix the resource/kind with the corresponding API group correct?  

This will make things a bit harder for us as it'll require some remembering / lookups.